### PR TITLE
Fix/Add MCSample Lines

### PR DESCRIPTION
**Pull request by JHiltbrand**
--Fixed and/or added entries to the MCSamples files to match with samples already present in the relevant sample dictionaries.
--Consequently, these samples were ntuplelized with unity event weight as event number information could not have been extracted.

- Summer20UL16.QCD_HT300to500: the `madgraphMLM` version is used, so `madgraph` version is updated in MCSamples
- Summer20UL16APV.QCD_HT200to300: use `madgraph` version in MCSamples (also incorrectly referenced non-APV files)
- Summer20UL16APV.QCD_HT300to500: correctly use APV and *not* non-APV files.
- Summer20UL17.TTToHadronic: simply missing from MCSamples, already present in dictionary
- Summer20UL18.TTToHadronic_hdampDOWN: simply missing from MCSamples, already present in dictionary

### CODE REVIEW

File: WeightProducer/python/MCSamples_Summer20UL16APV.py [hunk 1]
```diff
@@ -90,6 +90,7 @@
     MCSample('WJetsToQQ_HT-800toInf_TuneCP5_13TeV-madgraphMLM-pythia8', '106X_mcRun2_asymptotic_preVFP_v11-v2', 'RunIISummer20UL16MiniAODAPVv2', 'Constant', 7740501),
     MCSample('QCD_HT1000to1500_TuneCP5_PSWeights_13TeV-madgraphMLM-pythia8', '106X_mcRun2_asymptotic_preVFP_v11-v1', 'RunIISummer20UL16MiniAODAPVv2', 'Constant', 13823774),
     MCSample('QCD_HT100to200_TuneCP5_PSWeights_13TeV-madgraphMLM-pythia8', '106X_mcRun2_asymptotic_preVFP_v11-v1', 'RunIISummer20UL16MiniAODAPVv2', 'Constant', 67431856),
+    MCSample('QCD_HT200to300_TuneCP5_PSWeights_13TeV-madgraph-pythia8', '106X_mcRun2_asymptotic_preVFP_v11-v1', 'RunIISummer20UL16MiniAODAPVv2', 'Constant', 18273591),
```
Comment (aperloff) [inline WeightProducer/python/MCSamples_Summer20UL16APV.py:93]: HT300to500 seems to be available now as well. You might as well add that sample in this PR.

Comment (JHiltbrand) [reply WeightProducer/python/MCSamples_Summer20UL16APV.py:93]: Hi @aperloff ,
Sure I can get that in. Besides these fixes, I can merge in my other branch here, which takes care of samples missing from dictionaries/new samples. I was not sure if we wanted to keep this branch just for quick fixes.

Comment (aperloff) [reply WeightProducer/python/MCSamples_Summer20UL16APV.py:93]: Ummm...maybe just keep the PRs separate. It's easier to peel back the history is changes are small and well contained.

Comment (JHiltbrand) [reply WeightProducer/python/MCSamples_Summer20UL16APV.py:93]: Yep, totally fine with me. I did just add in the HT300to500 sample.

**Pull request merged**